src/confighttp.cpp

```diff
@@ -725,8 +736,7 @@ void start() {
   server.resource["^/api/clients/unpair$"]["POST"]         = unpairAll;
   server.resource["^/api/apps/close$"]["POST"]             = closeApp;
   server.resource["^/api/covers/upload$"]["POST"]          = uploadCover;
-  server.resource["^/images/favicon.ico$"]["GET"]          = getFaviconImage;
-  server.resource["^/images/logo-sunshine-45.png$"]["GET"] = getSunshineLogoImage;
+  server.resource["^/images\\/.+$"]["GET"]                 = getImages;
```

Comment:
Instead of basically duplicating the `getNodeModules` function, we should change the regex.

this should work:
```cpp
server.resource["^/(node_modules)|(images)\\/.+$"]["GET"] = getServerAsset;
```

Note that I renamed the function, which of course will need to get updated as well.

Reply:
Having something that could process files from the base directory would definitely be more appropriate.

I’m just not super confident with C++ and took the easier route.